### ISSUE TITLE
ROS2 VERSION

### ISSUE THREAD

**Issue by oscarlov** [D1Vincenzo/object_detection_in_gazebo_ros2#1]
Hi, what version of ros2 did you use? Because with ros2 Humble I can't open gazebo, I always have this error:

After going through step by step process i finallu ran this command
"rosdep install --from-paths src --ignore-src --rosdistro humble -r -y"

and i got this error
aiot@ubuntu:~/ros2_ws$ rosdep install --from-paths src --ignore-src --rosdistro humble -r -y
executing command [sudo -H apt-get install -y ros-humble-gazebo-ros2-control]
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package ros-humble-gazebo-ros2-control
executing command [sudo -H apt-get install -y ros-humble-gazebo-ros-pkgs]
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package ros-humble-gazebo-ros-pkgs
ERROR: the following rosdeps failed to install
apt: command [sudo -H apt-get install -y ros-humble-gazebo-ros2-control] failed
apt: command [sudo -H apt-get install -y ros-humble-gazebo-ros-pkgs] failed
apt: Failed to detect successful installation of [ros-humble-gazebo-ros2-control]
apt: Failed to detect successful installation of [ros-humble-gazebo-ros-pkgs]

**Comment by D1Vincenzo** [comment]
Hello! I am actually new to ros2 and still studying it, but I hope I can help to fix your problem. To answer your question, I am using ROS2 Humble too. Can I ask if you have installed gazebo correctly and where did you get this `rosdep install --from-paths src --ignore-src --rosdistro humble -r -y` command from?

**Comment by oscarlov** [comment]
No, the problem is actually that I can-t install Gazebo, infact for example if I try to install gazebo_ros_pkgs from this site https://classic.gazebosim.org/tutorials?tut=ros2_installing it return me the error that I wrote before. How did you install Gazebo Classic with ROS2 Humble? Thank you very much for helping me:)

**Comment by D1Vincenzo** [comment]
> No, the problem is actually that I can-t install Gazebo, infact for example if I try to install gazebo_ros_pkgs from this site https://classic.gazebosim.org/tutorials?tut=ros2_installing it return me the error that I wrote before. How did you install Gazebo Classic with ROS2 Humble? Thank you very much for helping me:)

I used exactly the command from the website you referenced: `sudo apt install ros-humble-gazebo-ros-pkgs`
As far as I know, different architecture of your computer's chip will lead to error while installing gazebo. Are you using a computer in arm64 chip such as virtual machine on mac?

**Comment by oscarlov** [comment]
I-m using a Raspberry Pi 4B

**Comment by D1Vincenzo** [comment]
> I-m using a Raspberry Pi 4B

So here is the problem, Raspberry Pi is based on ARM arch. If you have to use it for gazebo, try ubuntu 20.04 instead of 22.04 and the problem miget solved. I havent found any way to install gazebo on ARM based computer with ubuntu22.04 as well.
